charts/pega/charts/installer/config/prlog4j2.xml

```diff
@@ -88,8 +88,18 @@
 				<SizeBasedTriggeringPolicy size="250 MB"/>
 			</Policies>
 			<DefaultRolloverStrategy max="20"/>
-		</RollingRandomAccessFile> 
-		
+		</RollingRandomAccessFile>
+
+		<!-- RollingFile Appender for PegaCLUSTER logs -->
+		<RollingRandomAccessFile  name="CLUSTER" fileName="${sys:pega.logdir}/PegaCLUSTER.log" filePattern="${sys:pega.logdir}/PegaCLUSTER-%d{MM-dd-yyyy}-%i.log.gz">
+			<PatternLayout>
+				<Pattern>%d [%20.20t] [%20.20X{tenantid}] [%20.20X{app}] (%30.30c{3}) %-5p %X{stack} %X{userid} - %m%n</Pattern>
```

Comment:
Same with the Pattern, better to be in synch with other appender patterns. Tenantid may not make sense during installer.